src/models/AbstractStyledComponent.js

```diff
@@ -16,4 +17,5 @@ export default class AbstractStyledComponent extends Component {
 
 AbstractStyledComponent.contextTypes = {
   [CHANNEL]: PropTypes.func,
+  StyleSheet: PropTypes.instanceOf(StyleSheet),
```

Comment:
Do we want to make this more unique to avoid collisions with other libraries/apps?